eplaunch/interface/workflow_directories_dialog.py

```diff
@@ -1,11 +1,35 @@
 import wx
 
+from eplaunch.interface.crossplatform import Platform
 
-class WorkflowDirectoriesDialog(wx.Dialog):
 
+class WorkflowDirectoriesDialog(wx.Dialog):
     CLOSE_SIGNAL_OK = 0
     CLOSE_SIGNAL_CANCEL = 1
 
+    SampleWorkflows = {
+        Platform.WINDOWS: [
+            'c:\EnergyPlusV8-6-0\workflows',
+            'c:\EnergyPlusV8-7-0\workflows',
+            'c:\EnergyPlusV8-8-0\workflows',
+            'c:\EnergyPlusV8-9-0\workflows'
+        ],
+        Platform.LINUX: [
+            '/usr/local/bin/EnergyPlusV8-6-0/workflows',
+            '/usr/local/bin/EnergyPlusV8-7-0/workflows',
+            '/usr/local/bin/EnergyPlusV8-8-0/workflows',
+            '/usr/local/bin/EnergyPlusV8-9-0/workflows'
+        ],
+        Platform.MAC: [
+            '/Applications/EnergyPlusV8-6-0/workflows',
+            '/Applications/EnergyPlusV8-7-0/workflows',
+            '/Applications/EnergyPlusV8-8-0/workflows',
+            '/Applications/EnergyPlusV8-9-0/workflows'
+        ],
+        Platform.UNKNOWN: [
+        ]
+    }
+
```

Comment:
This will be generalized further, but we will indeed still want a set of predefined potential workflow directories.  Maybe once this is all said and done we could create workflow folders for older E+ versions that users could drop into old E+ installs... :+1:

Reply:
Why not?